pvlib/irradiance.py

```diff
@@ -793,20 +790,35 @@ def haydavies(surface_tilt, surface_azimuth, dhi, dni, dni_extra,
 
     Notes
     ------
+    The expression for the diffuse irradiance, :math:`I_d`, in the Hay and
+    Davies sky diffuse model is as follows:
+
+    .. math::
+            I_{d} = DHI \left( A\cdot R_b + (1 - A) \left(\frac{
+                1 + \cos\theta_T}{2}\right ) \right ).
```

Comment:
I am -0.5 to changing the symbol for surface tilt from $\beta$ to $\theta_T$ .  In the wild west of PV symbols, $\beta$ for surface tilt (and for angles up from the horizon generally) is at least somewhat widespread.  It's also the symbol used in the Loutzenhiser reference.